package.json

```diff
@@ -4,10 +4,11 @@
   "description": "A Javascript library for parsing metadata in a Web Page.",
   "main": "parser.js",
   "scripts": {
-    "test:lint": "./node_modules/eslint/bin/eslint.js ./*.js ./tests/*.js",
-    "test:mocha": "./node_modules/istanbul/lib/cli.js cover _mocha --report lcovonly -- tests/* -R spec",
-    "test": "./node_modules/npm-run-all/bin/npm-run-all/index.js test:*",
-    "cover": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
+    "test:lint": "eslint ./*.js ./tests/*.js",
```

Comment:
:+1: 
You could probably optimize this to just `"eslint ."` (and we can use .eslintignore to ignore any files we may not care about).